be/src/exec/parquet/file_reader.cpp

```diff
@@ -102,7 +102,8 @@ Status FileReader::_parse_footer() {
 
     tparquet::FileMetaData t_metadata;
     // deserialize footer
-    RETURN_IF_ERROR(deserialize_thrift_msg(footer_buf + to_read - 8 - footer_size, &footer_size, true, &t_metadata));
+    RETURN_IF_ERROR(deserialize_thrift_msg(footer_buf + to_read - 8 - footer_size, &footer_size, TProtocolType::COMPACT,
```

Comment:
add some comment to explain the meaning of TProtocolType::COMPACT

Reply:
Add comment to the definition of enum TProtocolType